### PR TITLE
📘 Read.me – Adding content to improve F5 clarity, and project structure details

### PR DESCRIPTION
**Pull request by isaaclombardssw**
Self-explanatory, read the readme 😄 

<details>
### General Contributing:

- [ ] Have you followed the guidelines in our [Contributing document](https://github.com/tinacms/tina.io/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### All New Content Submissions: (To be confirmed by reviewer)

- [ ] Title is short & specific
- [ ] Headers are logically ordered & consistent
- [ ] Purpose of document is explained in the first paragraph
- [ ] Procedures are tested and work
- [ ] Any technical concepts are explained or linked to
- [ ] Document follows structure from templates
- [ ] All links work
- [ ] The spelling and grammar checker has been run
- [ ] Graphics and images are clear and useful
- [ ] Any prerequisites and next steps are defined.
</details>

### CODE REVIEW

**Comment by isaaclombardssw** [comment]
@bradystroud as per our convo, move the little diagram to a `_docs/` directory (link it from the readme) and add a definition of ready / definition of done document as well

**Comment by isaaclombardssw** [comment]
Created https://github.com/tinacms/tina.io/issues/2251 for removing old guides stuff.

File: README.md [hunk 1]
```diff
@@ -1,40 +1,80 @@
...
+1. **Code Quality**:
+   - Use Tailwind over inline CSS or styled components.
+   - Follow DRY principles and refactor reusable code.
+   - Comment on calculations or workarounds, and open issues for any technical debt.
+1. **Pull Requests**:
+   - Reference related issues and include screenshots for UI changes.
+   - If no issue exists, create one and link it.
+   - Commit the generated `tina-lock.json` file, and rerun the project locally after merging to update it.
+1. **Review Process**:
+   - Due to Vercel deployment issues, maintainers will recreate your changes on a new branch to test before approval.
```
Comment (bradystroud) [inline README.md:69]: Move this stuff to contributing.md
as per github docs
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#adding-a-contributing-file

**Review by bradystroud — CHANGES_REQUESTED** [review]

**Review by bradystroud — APPROVED** [review]
LGTM
I'd make sure Gert agrees with DoD

File: _docs/Definition-of-Done.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Definition Of Done
+
+An issue in this repository can be closed (considered done) for...
+
+### Code + Content Changes
+
+1. Code compiles and change makes sense for the issue. All issue tasks have been ticked.
```
Comment (Marxoz) [inline _docs/Definition-of-Done.md:7]:
```suggestion
1. Code compiles and the change aligns with the issue description. All tasks related to the issue have been completed.
```

File: _docs/Definition-of-Done.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+1. When the latest commit has been manually tested against a preview deployment.
+1. When the code has been reviewed by a project maintainer.
```
Comment (Marxoz) [inline _docs/Definition-of-Done.md:9]:
```suggestion
1. The code has been reviewed and approved by a project maintainer.
```

File: _docs/Definition-of-Done.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+1. Code has been deployed to production.
+1. Any bugs or technical debt that arise from the issue have been logged in a PBI that links to this code change.
+1. Issue had been closed with a closing comment added describing solution or linking changed pages.
```
Comment (Marxoz) [inline _docs/Definition-of-Done.md:13]:
```suggestion
1. Issue had been closed with a closing comment added describing solution or linking changed pages.
1. All relevant automated tests have been executed and passed.
```

**Review by Marxoz — COMMENTED** [review]
Some suggestions

**Pull request merged**